roles/kubeadm_configs_compute/vars/main.yml

```
@@ -1,6 +1,11 @@
 ---
 default_kubeadm_config:
-  apiVersion: kubeadm.k8s.io/v1beta2
+  apiVersion: >-
+    {%- if _target_kube_version -%}
```

Didn't it miss the version comparison here?
```suggestion
    {%- if _target_kube_version is version("1.22", ">=") -%}
```

---

Lol indeed